nupm.nuon

```diff
@@ -1,7 +1,7 @@
-{
-    name: nupm
-    type: module
-    version: "0.2.0"
-    description: "Nushell package manager"
-    license: "LICENSE"
+
+    name: nupm,
```

Comment:
These changes to nupm.nuon aren't necessary

Reply:
@kubouch is there a spec for nuon format? I thought it was a superset of json, I looked here and could not find a bnf or spec documentation other than test cases:
https://github.com/nushell/nushell/tree/main/crates/nuon